docs/code_ref/coordinates/carrington.rst

```diff
@@ -0,0 +1,126 @@
+
+|AA| publishes the apparent sub-Earth Carrington longitude (|L0|), and these values include the stellar-aberration correction.
+Consequently, SunPy values will be consistently ~0.006 degrees (~20 arcseconds) greater than |AA| values, although these discrepancies are not always apparent at the printed precision (0.01 degrees).
+
+Since |AA| specifically tuned the IAU parameters to minimize the discrepancies with older methods of calulation under their prescription that includes the stellar-aberration correction, SunPy values will also be ~20 arcseconds greater than values calculated using older methods.
+Be aware that older methods of calculation may not have accounted for the variable light travel time between the Sun and the Earth, which can cause additional discrepancies of up to ~5 arcseconds.
+
+|AA| does not appear to account for the difference in light travel time between the sub-Earth point and the center of the Sun (~2.3 lightseconds), which results in a fixed discrepancy of ~1.4 arcseconds in |L0|.
+However, this additional discrepancy is much smaller than the difference in treatment of stellar aberration.
+
+Compared to SPICE
+^^^^^^^^^^^^^^^^^
+
+In `SPICE <https://naif.jpl.nasa.gov/naif/>`_, the apparent sub-observer Carrington longitude can be calculated using the function `subpnt <https://naif.jpl.nasa.gov/pub/naif/toolkit_docs/C/cspice/subpnt_c.html>`_.
+`subpnt`_ allows for two types of aberration correction through the input argument ``abcorr``:
+
+* "LT", which is just the correction for light travel time
+* "LT+S", which is the correction for light travel time ("LT") plus the correction for observer motion ("S")
+
+SunPy calculates the apparent sub-observer Carrington longitude in a manner equivalent to specifying "LT" (as opposed to "LT+S").
+The discrepancy between SunPy values and SPICE values is no greater than 0.01 arcseconds.
+
+Compared to JPL Horizons
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+In `JPL Horizons <https://ssd.jpl.nasa.gov/?horizons>`_, one can request the "Obs sub-long & sub-lat".
+JPL Horizons appears to start from the IAU parameters and to include the correction for light travel time but not the correction for observer motion (i.e., equivalent to specifying "LT" to `subpnt`_ in `SPICE`_).
+The discrepancy between SunPy values and JPL Horizons values is no greater than 0.01 arcseconds.
+
+Compared to SunSPICE
+^^^^^^^^^^^^^^^^^^^^
+
+In `SunSPICE <https://stereo-ssc.nascom.nasa.gov/sunspice.shtml>`_, one can convert to and from the "Carrington" coordinate system using the function `convert_sunspice_coord <https://hesperia.gsfc.nasa.gov/ssw/packages/sunspice/idl/convert_sunspice_coord.pro>`_.
+However, these Carrington longitudes are "true" rather than "apparent" because the observer is not specified, so there are no corrections for light travel time or for observer motion.
+Thus, SunPy values will be consistently greater than SunSPICE's coordinate transformation by ~0.82 degrees.
```

Comment:
Also, for all the comparisons. Are these discrepancies only in longitude, in latitude or combined?

Reply:
The short answer is that we agree in latitude with JPL Horizons and SPICE.  *The Astronomical Almanac* does the same thing with applying a stellar-aberration correction, so that causes us to disagree with them up to a few arcseconds, but that would almost never show up at their printed precision.